src/main/scala/slamdata/SbtSlamData.scala

```diff
@@ -188,7 +188,7 @@ object SbtSlamData extends AutoPlugin {
   import autoImport._
 
   override def globalSettings = Seq(
-    concurrentRestrictions := {
+    concurrentRestrictions in Global := {
```

Comment:
So you added all the `in Global` back?

Reply:
Yeah. :-( I'm just being safe on `concurrentRestrictions`, even though I'm pretty sure it worked the way you had it. `useGpg in Global` is the important one, since sbt-pgp does its scoping incorrectly.